pandas/core/series.py

```diff
@@ -2993,10 +2993,13 @@ def argsort(self, axis=0, kind='quicksort', order=None):
         Parameters
         ----------
         axis : int (can only be zero)
```

Comment:
The "(can only be zero)" can be removed, as it doesn't appear that this is checked:
```python
In [2]: s = pd.Series([3, 1, 4, 1, 5])

In [3]: s.argsort(axis=10)
Out[3]:
0    1
1    3
2    0
3    2
4    4
dtype: int64
```